project/profiles/capacity4more/modules/c4m/user/c4m_user/c4m_user.module

```diff
@@ -1286,3 +1299,46 @@ function c4m_user_ckeditor_settings_alter(&$settings, $conf) {
     unset($settings['loadPlugins']['media']);
   }
 }
+
+/**
+ * Implements hook_action_info_alter().
+ *
+ * Removes 'contact multiple' action, if user is not admin / group owner.
+ */
+function c4m_user_action_info_alter(&$actions) {
```

Comment:
Remove this function and add the logic to `function c4m_user_contact_multiple_action_info()` on line 14